programs/perpetuals/src/state/pool.rs

```diff
@@ -642,12 +589,50 @@ impl Pool {
                 custody.oracle.max_price_error,
                 custody.oracle.max_price_age_sec,
                 curtime,
+                false,
             )?;
 
             let token_amount_usd =
                 token_price.get_asset_amount_usd(custody.assets.owned, custody.decimals)?;
 
             pool_amount_usd = math::checked_add(pool_amount_usd, token_amount_usd as u128)?;
+
+            if custody.pricing.use_unrealized_pnl_in_aum {
```

Comment:
can use `is_stable` to decide on this?

Reply:
I was thinking to have an additional parameter to make it possible to disable unrealized pnl in AUM calculations for highly volatile assets like NFTs, but probably it wouldn't serve the purpose anyway, so will get it removed